wlauto/workloads/googlephotos/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java

```diff
@@ -27,24 +28,15 @@
     public void runUiAutomation() throws Exception {
         parameters = getParams();
 
-        confirmLocalFileAccess();
         dismissWelcomeView();
         gesturesTest();
-        editPhotoTest();
+        editPhotoColorTest();
+        cropPhotoTest();
+        rotatePhotoTest();
 
         writeResultsToFile(timingResults, parameters.getString("output_file"));
     }
 
-    private void confirmLocalFileAccess() throws Exception {
-        // First time run requires confirmation to allow access to local files
-        UiObject allowButton = new UiObject(new UiSelector().textContains("Allow")
-                                                            .className("android.widget.Button"));
-        if (allowButton.waitForExists(timeout)) {
-            allowButton.clickAndWaitForNewWindow(timeout);
-        }
-    }
-
-
```

Comment:
It could but how do we automate that step?  There should be no need to touch the target device when running any of workloads.  Skype is even stranger.  On two devices you just need to click once.   On another it needs to click 5 times (access to file storage, microphone, camera, etc, etc)!